### PR TITLE
Victor cm006

### PR DESCRIPTION
**Pull request by wvictor14**
- removed metaprogramming section
- added `mutate`  to content
- rearranged: filter to after the break
- added partially filled code for most exercises
- added/modified 2-3 exercises

I thought adding in `group_by` content would be too much content to cover, but I'm not sure so let me know what you think?

Also wondering if there is content that I can look at for the relational/logical operators section?

### CODE REVIEW

File: cm006.Rmd [hunk 1]
```diff
@@ -22,11 +22,11 @@ Here are some supplementary resources:
 
 ## Participation
 
-To get participation points for today, we'll be filling out the [cm005-exercise.Rmd](https://github.com/STAT545-UBC/Classroom/blob/master/notes/cm005-exercise.Rmd) file, and adding it to your participation repo.
+To get participation points for today, we'll be filling out the [cm006-exercise.Rmd](https://github.com/STAT545-UBC/Classroom/blob/master/tutorials/cm006-exercise.Rmd) file, and adding it to your participation repo.
 
 Let's get set up:
 
-1. I made it easier to download this time: just click the upper-right drop-down menu [of the html version](http://stat545.com/Classroom/notes/cm005-exercise.nb.html).
+1. I made it easier to download this time: just click the upper-right drop-down menu [of the html version](http://stat545.com/Classroom/notes/cm006-exercise.nb.html).
```
Comment (vincenzocoia) [inline cm006.Rmd]: the "this time" comment really only makes sense in the context of last year.

File: tutorials/cm006-exercise.Rmd [hunk 2]
```diff
@@ -13,39 +18,154 @@ Change the file output to both html and md _documents_ (not notebook).
 
 1. Load the `gapminder` and `tidyverse` packages. Hint: `suppressPackageStartupMessages()`!
     - This loads `dplyr`, too.
+    
+```{r load packages, warning = F, message = F}
```
Comment (vincenzocoia) [inline tutorials/cm006-exercise.Rmd]: Would prefer to use `FALSE` instead of `F`. Just better practice that I think we should demonstrate.

**Review by vincenzocoia — CHANGES_REQUESTED** [review]
Looks great! Just a couple of changes. See the line-specific comments.

**Comment by vincenzocoia** [comment]
> Also wondering if there is content that I can look at for the relational/logical operators section?

Are you looking for a resource for them to read more about this? If so, probably the "Advanced R" or "R for data science" books. Although, we did cover these in class on Day 2, so they should know at least a bit about them.

File: tutorials/cm006-exercise.Rmd (hunk 2 above)
Comment (wvictor14) [reply tutorials/cm006-exercise.Rmd]: Sounds good

**Comment by wvictor14** [comment]
> > Also wondering if there is content that I can look at for the relational/logical operators section?
> 
> Are you looking for a resource for them to read more about this? If so, probably the "Advanced R" or "R for data science" books. Although, we did cover these in class on Day 2, so they should know at least a bit about them.

I think my plan will be to just add a table of the operators in the guide and then briefly discuss them before showing them out in the exercises.

**Comment by wvictor14** [comment]
I think those are all the changes you requested. Let me know if there's anything else! And, should I be worried about these failed netlify checks? I can't view the logs for some reason.

**Pull request merged**